### PR TITLE
Contants and translations

### PR DESCRIPTION
**Pull request by razvanbugariu**
Niciun ticket. CONSTIINTA!

### CODE REVIEW

File: app/components/authorization/authorization.service.js [hunk 1]
```diff
@@ -2,46 +2,46 @@
...
-        if (this.isLoggedIn() && userRoles.indexOf($cookies.get("userRole")) !== -1) {
+        if (this.isLoggedIn() && userRoles.indexOf($cookies.get(Constants.USER_ROLE)) !== -1) {
           return true;
         }
         return false;
       }
 
       function isNotEmpty(element) {
-        if (element !== "") {
+        if (element !== Constants.EMPTY) {
```
Comment (gimmyxd) [inline app/components/authorization/authorization.service.js:16]: I Don't think we need constant here. Also you can put it like: `return element !== ''`

File: app/components/authorization/authorization.service.js [hunk 2]
```diff
@@ -2,46 +2,46 @@
 
 angular
   .module('mentors4me')
-  .factory('authorizationService', function($cookies, USER_ROLES) {
+  .factory('authorizationService', function($cookies, USER_ROLES, Constants) {
       var authorizationService = {};
 
       authorizationService.isAuthorized = function(userRoles) {
-        if (this.isLoggedIn() && userRoles.indexOf($cookies.get("userRole")) !== -1) {
+        if (this.isLoggedIn() && userRoles.indexOf($cookies.get(Constants.USER_ROLE)) !== -1) {
```
Comment (gimmyxd) [inline app/components/authorization/authorization.service.js:9]: You can use early return with the result of the block from the if

File: app/components/authorization/authorization.service.js [hunk 3]
```diff
@@ -2,46 +2,46 @@
...
-        if (element !== "") {
+        if (element !== Constants.EMPTY) {
           return true;
         }
         return false;
       }
 
       authorizationService.isLoggedIn = function() {
-        if (!angular.isUndefined($cookies.get("token")) && isNotEmpty($cookies.token)) {
+        if (!angular.isUndefined($cookies.get(Constants.TOKEN)) && isNotEmpty($cookies.token)) {
```
Comment (gimmyxd) [inline app/components/authorization/authorization.service.js:23]: You can use early return with the result of the block from the if

File: app/components/authorization/authorization.service.js [hunk 4]
```diff
@@ -2,46 +2,46 @@
...
           return true;
         }
         return false;
       }
 
       authorizationService.isMentor = function() {
-        if (!angular.isUndefined($cookies.get("token")) && isNotEmpty($cookies.get("userRole")) && $cookies.get("userRole") === USER_ROLES.MENTOR) {
+        if (!angular.isUndefined($cookies.get(Constants.TOKEN)) && isNotEmpty($cookies.get(Constants.USER_ROLE)) && $cookies.get(Constants.USER_ROLE) === USER_ROLES.MENTOR) {
```
Comment (gimmyxd) [inline app/components/authorization/authorization.service.js:30]: You can use early return with the result of the block from the if

File: app/components/authorization/authorization.service.js [hunk 5]
```diff
@@ -2,46 +2,46 @@
...
           return true;
         }
         return false;
       }
 
       authorizationService.isAdmin = function() {
-        if (!angular.isUndefined($cookies.get("token")) && isNotEmpty($cookies.get("userRole")) && $cookies.get("userRole") === USER_ROLES.ADMIN) {
+        if (!angular.isUndefined($cookies.get(Constants.TOKEN)) && isNotEmpty($cookies.get(Constants.USER_ROLE)) && $cookies.get(Constants.USER_ROLE) === USER_ROLES.ADMIN) {
```
Comment (gimmyxd) [inline app/components/authorization/authorization.service.js:37]: You can use early return with the result of the block from the if

File: app/components/authorization/authorization.service.js [hunk 6]
```diff
@@ -2,46 +2,46 @@
...
           return true;
         }
         return false;
       }
 
       authorizationService.isOrganization = function() {
-        if (!angular.isUndefined($cookies.get("token")) && isNotEmpty($cookies.get("userRole")) && $cookies.get("userRole") === USER_ROLES.ORGANIZATION) {
+        if (!angular.isUndefined($cookies.get(Constants.USER_ROLE)) && isNotEmpty($cookies.get(Constants.USER_ROLE)) && $cookies.get(Constants.USER_ROLE) === USER_ROLES.ORGANIZATION) {
```
Comment (gimmyxd) [inline app/components/authorization/authorization.service.js:44]: You can use early return with the result of the block from the if

File: app/components/register/register.controller.js [hunk 7]
```diff
@@ -25,7 +25,8 @@ function registerController($scope, $location, registerService, growl) {
   }
 
   function handleCreateSuccess(response){
-		growl.info("Contul dumneavoastra a fost creat cu succes!");
```
Comment (gimmyxd) [inline app/components/register/register.controller.js:28]: indentation

Comment (razvanbugariu) [reply app/components/register/register.controller.js:28]: This line was deleted...

**Review by gimmyxd — APPROVED** [review]

**Pull request merged**